play-2.4/swagger-play2/app/play/modules/swagger/SwaggerPlugin.scala

```diff
@@ -81,9 +82,9 @@ class SwaggerPluginImpl @Inject()(lifecycle: ApplicationLifecycle, router: Route
   }
 
   val routesFile = config.underlying.hasPath("play.http.router") match {
-    case false => "routes"
+    case false => "conf/routes"
     case true => config.getString("play.http.router") match {
-      case None => "routes"
+      case None => "conf/routes"
       case Some(value)=> value.replace(".Routes", ".routes")
```

Comment:
Yes - so the `Some` case also needs "conf/" adding to the end so it will find custom routes files inside conf